addons/actions/src/preview/withActions.ts

```diff
@@ -56,11 +56,11 @@ const actionsSubscription = (...args: any[]) => {
   return lastSubscription;
 };
 
-export const createDecorator = (actionsFn: any) => (...args: any[]) => (story: () => any) => {
+export const createDecorator = (actionsFn: any) => (...args: any[]) => (fn: () => any) => {
```

Comment:
Is `fn` a bit unclear - I think `storyFn()` makes more sense, no?

Reply:
Yea I don't feel strongly about that. I do feel `fn` is better then `story`, but I don't really prefer `fn` over `storyFn` or vice-versa. I picked `fn` since it's a little less verbose.

If you really feel `stroyFn` > `fn`, I'll change it.

Reply:
I think it is best to be consistent and clear. So I vote for `storyFn` everywhere